packages/ant-component-mapper/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@data-driven-forms/ant-component-mapper",
+  "version": "0.0.5",
+  "description": "Component mapper for Ant Design component mapper form data-driven-forms.",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "license": "Apache-2.0",
+  "scripts": {
+    "start": "webpack-dev-server --env dev --config ./config/webpack.config.js --open --hot",
+    "build": "rollup -c ./rollup.config.js",
+    "vendor": "webpack --env vendor --config ./config/webpack.config.js",
+    "release": "semantic-release"
+  },
+  "repository": "git@github.com:data-driven-forms/react-forms.git",
+  "devDependencies": {
+    "@ant-design/icons": "^4.0.2",
+    "@babel/core": "^7.1.6",
+    "@babel/plugin-proposal-class-properties": "^7.2.3",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/preset-env": "^7.1.6",
+    "@babel/preset-react": "^7.0.0",
+    "@data-driven-forms/react-form-renderer": "*",
+    "@semantic-release/git": "^7.0.5",
+    "@semantic-release/npm": "^5.1.1",
+    "antd": "^4.2.0",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-eslint": "9.x",
+    "babel-jest": "^23.6.0",
+    "babel-loader": "^8.0.4",
+    "babel-plugin-lodash": "^3.3.4",
+    "css-loader": "^1.0.1",
+    "enzyme": "^3.7.0",
+    "enzyme-adapter-react-16": "^1.7.0",
+    "enzyme-to-json": "^3.3.4",
+    "glob": "^7.1.6",
+    "html-webpack-plugin": "^3.2.0",
+    "mini-css-extract-plugin": "^0.4.4",
+    "node-sass": "^4.10.0",
+    "prop-types": "^15.6.2",
+    "react": "^16.13.0",
+    "react-dom": "^16.13.0",
+    "regenerator-runtime": "^0.12.1",
+    "resolve-url-loader": "^3.0.0",
+    "rollup-plugin-sourcemaps": "^0.5.0",
+    "rollup-pluginutils": "^2.8.2",
+    "sass-loader": "^7.1.0",
+    "semantic-release": "15.12.0",
+    "style-loader": "^0.23.1",
+    "url-loader": "^1.1.2",
+    "webpack": "^4.25.1",
+    "webpack-cli": "^3.1.2",
+    "webpack-dev-server": "^3.1.10",
+    "webpack-merge": "^4.1.4"
+  },
+  "peerDependencies": {
+    "@data-driven-forms/react-form-renderer": "^1.9.3",
+    "react": "^16.13.0",
+    "react-dom": "^16.13.0"
+  },
+  "release": {
+    "prepare": [
+      "@semantic-release/npm",
+      {
+        "path": "@semantic-release/git",
+        "assets": [
+          "package.json"
+        ],
+        "message": "Release of new version: ${nextRelease.version} <no> [skip ci]"
+      }
+    ]
+  },
+  "dependencies": {
+    "@date-io/date-fns": "^1.0.1",
+    "@date-io/moment": "^1.0.1",
+    "classnames": "^2.2.6",
+    "clsx": "^1.0.4",
+    "date-fns": "^1.30.1",
+    "moment": "^2.23.0",
+    "react-select": "^2.2.0"
```

Comment:
I don't see any of these deps being used here. If they are needed by the ant library, please include them in dev & peer deps.